### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by VeryOldLady**
authentification added

### CODE REVIEW

File: src/services/auth/auth.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+func Login(w http.ResponseWriter, r *http.Request) {
+
+	var newLogin login
+
+	r.ParseForm()
+	newLogin.login = r.Form.Get("login")
+	newLogin.pass = r.Form.Get("password")
+
+	var usersInDB User
+	err := db.QueryRow("SELECT ID, Password FROM Users WHERE Login = $1", newLogin.login).Scan(&(usersInDB.ID), &(usersInDB.Password))
```
Comment (Nastya-Kruglikova) [inline src/services/auth/auth.go]: You should call methods from model that are already done

File: src/services/auth/auth.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	if newLogin.pass == usersInDB.Password {
+		sessionUUID, err := uuid.NewV1()
+		if err != nil {
+			fmt.Println("##### ", err)
+			common.SendError(w, r, 418, "ERROR: ", err)
```
Comment (Nastya-Kruglikova) [inline src/services/auth/auth.go]: Pls, use 401 (Unauthorized) instead of 418 (I'm a teapot)

File: src/services/auth/auth.go [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+	common.SendError(w, r, 418, "ERROR: ", errors.New("no result"))
+
+}
+
+func Logout(w http.ResponseWriter, r *http.Request) {
+	var newLogout login
+	r.ParseForm()
+	newLogout.sessionID = r.Form.Get("sessionID")
+	redis.Del(newLogout.sessionID)
+	common.RenderJSON(w, r, "")
```
Comment (Nastya-Kruglikova) [inline src/services/auth/auth.go]: Maybe "success logout" instead of ""

File: api/swagger.yaml [hunk 4]
```diff
@@ -22,6 +23,91 @@ paths:
           description: Hello World
           schema:
             $ref: '#/definitions/HelloWorldResponse'
+  /login:
```
Comment (mshto) [inline api/swagger.yaml]: Could you validate a swagger file here : https://editor.swagger.io/ ?

**Pull request merged**